CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 
 ## vN.N.N
+* Make only atomic blocks be advanced blocks
```

Comment:
@aloncohenn more description of why we are doing this, like the ticket name, would be good